executors.go

```diff
@@ -161,6 +291,22 @@ func (c *Connection) Update(model interface{}, excludeColumns ...string) error {
 
 // Destroy deletes a given entry from the database
 func (c *Connection) Destroy(model interface{}) error {
+	v := reflect.Indirect(reflect.ValueOf(model))
```

Comment:
I feel like a lot of this logic is repeated. would make sense to break it out into it's own method?